GridOutputContainerLayoutManager #4198

kpaharelau opened the pull request:
> should have a constructor with no args that defaults to 2 columns.

done

kpaharelau committed:
GridOutputContainerLayoutManager #4198

>should have a constructor with no args that defaults to 2 columns.

done

scottdraves commented on core/config/tutorials/output_container_cell.bkr:
```diff
@@ -14364,7 +14364,7 @@
             "input": {
                 "body": [
                     "def o = new OutputContainer()",
-                    "o.setLayoutManager(new GridOutputContainerLayoutManager(2)) // 2 columns",
```
please don't change the documentation

kpaharelau committed:
GridOutputContainerLayoutManager #4198

@scottdraves     Two Sigma Open Source member
scottdraves added a note an hour ago
please don't change the documentation

done
